FirebaseAppCheck/Sources/Public/FirebaseAppCheck/FIRAppCheck.h

```diff
@@ -38,6 +51,20 @@ NS_SWIFT_NAME(AppCheck)
 /// @throw Throws an exception if required `FirebaseApp` options are missing.
 + (nullable instancetype)appCheckWithApp:(FIRApp *)firebaseApp NS_SWIFT_NAME(appCheck(app:));
 
+/// Requests Firebase app check token. This method should *only* be used if you need to authorize
+/// requests to a non-Firebase backend. Requests to Firebase backend are authorized automatically if
+/// configured.
+/// @param forcingRefresh If `YES`,  a new Firebase app check token is requested and the token
+/// cache is ignored. If `NO`, the cached token is used if it exists and has not expired yet. In
+/// most cases, `NO` should be used. `YES` should only be used if the server explicitly returns an
+/// error, indicating a revoked token.
+/// @param handler The completion handler. Includes the app check token if the request succeeds,
+/// or an error if the request fails.
+- (void)appCheckTokenForcingRefresh:(BOOL)forcingRefresh
+                         completion:(void (^)(FIRAppCheckToken *_Nullable token,
+                                              NSError *_Nullable error))handler
+    NS_SWIFT_NAME(appCheckToken(forcingRefresh:completion:));
```

Comment:
Briefly discussed after API review but wanted to touch base about API naming. I've renamed this method from `getAppCheckTokenForcingRefresh` → `appCheckTokenForcingRefresh` to make it more swift async/await friendly without having to additionally specify the `NS_SWIFT_ASYNC_NAME`. I'm thinking the general guidance could be to remove the `get` prefix from any new APIs (if applicable). This comes from Apple's recommendation to remove the `get` prefix from async/await method names.

Swift Async/Await usage example:
```swift
let token = try await AppCheck.appCheck().appCheckToken(forcingRefresh: true)
```
Side note, since "app check" appears three times in the above call, I wonder if `token(forcingRefresh:)` is a better option. 
```swift
let token = try await AppCheck.appCheck().token(forcingRefresh: true)
```

Thoughts?

Reply:
As discussed offline, dropping `get` looks fine. Let's update the API proposal correspondingly and send a heads up to the API council.